objects/index.js

```diff
@@ -84,8 +84,8 @@ export const joinObjects = (objA, objB) => {
   isObject should also be set to true
 */
 export const addIsObjectProperty = (obj) => {
-  let newObj = {
+  const isObj = {
     isObject: true
   };
-  return newObj;
+  return Object.assign(obj, isObj);
```

Comment:
Nice, this is actually a second way of solving this problem: 

```js
const newObj = obj;
newObj.isObject = true;
return newObj
```

Reply:
We need to have a chat about whats going on with these functions too, the difference between modifying an existing object and creating a new one